Lib/ufo2ft/fontInfoData.py

```diff
@@ -311,8 +311,8 @@ def postscriptWeightNameFallback(info):
     openTypeVheaCaretOffset=None,
 
     postscriptUniqueID=None,
-    postscriptUnderlineThickness=None,
-    postscriptUnderlinePosition=None,
+    postscriptUnderlineThickness=50,
+    postscriptUnderlinePosition=-75,
```

Comment:
I'll note though that I don't think the FDK scales these by upm, as can be seen in Glyphs-produced binaries.